src/ts/order.ts

```diff
@@ -169,6 +179,21 @@ export function normalizeOrder(order: Order): NormalizedOrder {
   };
 }
 
+/**
+ * Compute the 32-byte signing hash for the specified order.
+ *
+ * @param domain The EIP-712 domain separator to compute the hash for.
+ * @param types The order to compute the digest for.
+ * @return Hex-encoded 32-byte order digest.
+ */
+export function hashTypedData(
+  domain: TypedDataDomain,
+  types: Record<string, TypedDataField[]>,
+  data: NormalizedOrder | OrderCancellation,
```

Comment:
nit: This should be `unknown` as we want it work for any type (that we don't have information for).
```suggestion
  data: unknown,
```

**edit**: It looks like the `hash` method requires a more specific type, so maybe use 
```suggestion
  data: Record<string, unknown>,
```